pkgs/development/python-modules/pymssql/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  freetds,
+  krb5,
+  openssl,
+  cython,
+  setuptools-scm,
+  sqlparse,
+  tomli,
+}:
+
+buildPythonPackage rec {
+  pname = "pymssql";
+  version = "2.3.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-8DTka1aAYdFxSPHe3qZI+dX2tzQOjP/g7bqhcTy0q6s=";
+  };
+
+  build-system = [
+    setuptools-scm
+    tomli
+    cython
+  ];
+
+  buildInputs = [
+    krb5
+    openssl
+    freetds
+  ];
+
+  propagatedBuildInputs = [ sqlparse ];
+
+  pythonImportsCheck = [ "pymssql" ];
+
+  meta = with lib; {
+    description = "A simple database interface for Python that builds on top of FreeTDS to provide a Python DB-API (PEP-249) interface to Microsoft SQL Server.";
```

```suggestion
    description = "Simple database interface for Python that builds on top of FreeTDS to provide a Python DB-API (PEP-249) interface to Microsoft SQL Server";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes